lib/client-api/src/story_store.js

```diff
@@ -89,7 +89,12 @@ export default class StoryStore extends EventEmitter {
     this._selection = data === undefined ? this._selection : { storyId, viewMode };
     this._error = error === undefined ? this._error : error;
 
-    setTimeout(() => this.emit(Events.STORY_RENDER), 1);
+    setTimeout(() => {
+      if (this._channel) {
```

Comment:
@ndelangen, I am not sure what kind of implications this could have.

Reply:
Should be fine, we want to migrate to using the channel for this anyway, might be good to add a code-comment there to know which is the preferred way.